src/extension/php_objects.c

```diff
@@ -60,13 +70,14 @@ _Static_assert(offsetof(zend_string, val) % _Alignof(struct entry_ex) == 0,
 void dd_phpobj_reg_ini_env(const dd_ini_setting *sett)
 {
     size_t name_len = NAME_PREFIX_LEN + sett->name_suff_len;
-    char *name = emalloc(name_len + 1);
+    char *name = malloc(name_len + 1);
```

Comment:
According to the documentation at MINIT the Zend Memory Manager and instead only permanent memory allocations should be done since there is no request yet